### PR TITLE
Coinjoin: add blame round

### PR DESCRIPTION
**Pull request by MarnixCroes**

### CODE REVIEW

File: docs/using-wasabi/CoinJoin.md [hunk 1]
```diff
@@ -127,6 +127,17 @@ Each Alice does the following:
 
 The signing phase ends when the coordinator has received all the valid signatures for all the registered inputs.
 
+### Blame round
+
+When the signing phase fails from becoming succesful duo to some Alices disrupting the round (failing to sign or send the signature to the coordinator), then the succesful Alices will continue into a blame round.
```
Comment (MaxHillebrand) [inline docs/using-wasabi/CoinJoin.md:132]:
```suggestion
When the signing phase fails from becoming succesful due to some Alices disrupting the round (failing to sign or send the signature to the coordinator), then the succesful Alices will continue into a blame round.
```

File: docs/using-wasabi/CoinJoin.md [hunk 2]
```diff
@@ -127,6 +127,17 @@ Each Alice does the following:
 
 The signing phase ends when the coordinator has received all the valid signatures for all the registered inputs.
 
+### Blame round
+
+When the signing phase fails from becoming succesful due to some Alices disrupting the round (failing to sign or send the signature to the coordinator), then the succesful Alices will continue into a blame round.
```
Comment (yahiheb) [inline docs/using-wasabi/CoinJoin.md:132]:
```suggestion
When the signing phase fails due to some Alices disrupting the round (failing to sign or send the signature to the coordinator), then the successful Alices will continue into a blame round.
```

File: docs/using-wasabi/CoinJoin.md [hunk 3]
```diff
@@ -127,6 +127,17 @@ Each Alice does the following:
...
+The blame round will redo the coinjoin phases in order to create a succesful coinjoin.
```
Comment (yahiheb) [inline docs/using-wasabi/CoinJoin.md:133]:
```suggestion
The blame round will redo the coinjoin phases in order to create a successful transaction.
```

File: docs/using-wasabi/CoinJoin.md [hunk 4]
```diff
@@ -127,6 +127,17 @@ Each Alice does the following:
...
+
+Only succesfully registered coins are allowed into the new blame round, so this also prevents coinjoins from being DDoS-ed.
```
Comment (yahiheb) [inline docs/using-wasabi/CoinJoin.md:135]:
```suggestion
Only successfully registered coins are allowed into the new blame round, so this also prevents coinjoins from being DDoS-ed.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Comment by yahiheb** [comment]
@turbolay What do you think about this one?

File: docs/using-wasabi/CoinJoin.md [hunk 5]
```diff
@@ -127,6 +127,17 @@ Each Alice does the following:
 
 The signing phase ends when the coordinator has received all the valid signatures for all the registered inputs.
 
+### Blame round
+
+When the signing phase fails due to some Alices disrupting the round (failing to sign or send the signature to the coordinator), then the successful Alices will continue into a blame round.
+The blame round will redo the coinjoin phases in order to create a successful coinjoin.
+
+Only successfully registered coins are allowed into the new blame round, so this also prevents coinjoins from being DDoS-ed.
+The client will keep going to the blame round until there is none.
```
[the remark this answers is not in the document]
Comment (MarnixCroes) [reply docs/using-wasabi/CoinJoin.md:136]: I copied the comment https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi/WabiSabi/Client/CoinJoinClient.cs#L172

**Comment by MarnixCroes** [comment]
> blame round" whatsoever, so whether we need to change the music box message

done that :), it was nacked
https://github.com/zkSNACKs/WalletWasabi/pull/9393

imo makes most sense to change the message in musicbox, as it's the only place where it's fallback

File: docs/using-wasabi/CoinJoin.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (MarnixCroes) [reply docs/using-wasabi/CoinJoin.md]: improved it, thanks

**Review by yahiheb — APPROVED** [review]

**Pull request merged**